packages/permission-controller/src/rpc-methods/requestPermissions.test.ts

```diff
@@ -99,7 +99,7 @@ describe('requestPermissions RPC method', () => {
       delete expectedError.stack;
 
       const response: any = await engine.handle(req as any);
-      expect(response.error).toStrictEqual(expectedError);
+      expect(response.error).toMatchObject(expectedError);
```

Comment:
Due to the fact that we use `~` to specify dependencies, and `yarn.lock` was effectively recreated, we are now using a slightly newer version of `eth-rpc-errors`, so that `response.error` now contains a `stack` property when it didn't before. I couldn't figure out why this was, as [this PR](https://github.com/MetaMask/eth-rpc-errors/pull/36) doesn't seem to make any breaking changes. I didn't want to waste any more time on this, but the stack isn't that important, so I quickly amended these tests so they worked again.

Reply:
We can undo both of these changes now; it was resolved with one of the dependency downgrades.

Reply:
Will do this in another PR.